dev_menu.py

```diff
@@ -99,21 +99,21 @@ def create_virtualenv(venv_exe, pyexe, venv) -> None:
         logging.warn("Skipping creation of virtualenv")
         return
     check_call([venv_exe, '-p', pyexe, venv])
-    activate_this_py = os.path.join(venv, 'bin', 'activate_this.py')
-    # Activate virtualenv in this interpreter
-    exec(open(activate_this_py).read(), dict(__file__=activate_this_py))
-    check_call(['pip', 'install', '--upgrade','--force-reinstall', '-e', 'python'])
-    check_call(['pip', 'install', '-r', 'tests/requirements.txt'])
+    # TODO: Activate virtualenv in this interpreter, this is not working as it is.
```

Comment:
This change seems to be unrelated to this PR. Can we make the PR focused and maybe create a separte PR for this change?

Reply:
Yes, is still marked as [Don't merge], it's WIP.